### ISSUE TITLE
Indentation missing in ordered list

### ISSUE THREAD

**Issue by sunidhiazad** [hlxsites/servicenow#141]
Original: https://www.servicenow.com/blogs/2022/microsoft-adopts-predictive-intelligence.html
EDS: https://main--aemeds--servicenow-martech.hlx.live/blogs/2022/microsoft-adopts-predictive-intelligence

Scroll down to **Key recommendations**

**Comment by sunidhiazad** [comment]
<p style="margin-left: 40.0px;"></p>
Styling is not imported.

**Comment by andreituicu** [comment]
If this is not a pattern that we see on multiple pages, I wouldn't fix it for a 1 off example. instead, I would fix this particular word document to use an ordered list, which has padding out of the box.

**Comment by sunidhiazad** [comment]
Yes, that makes sense. I only found it on one of the pages, but there might be more. I will keep this open either way to keep in mind to fix the document or fix in importer if seen in multiple pages.

**Comment by sunidhiazad** [comment]
https://main--aemeds--servicenow-martech.hlx.live/uk/blogs/2023/how-to-use-customer-profiling
https://www.servicenow.com/uk/blogs/2023/how-to-use-customer-profiling.html
